easybuild/easyblocks/generic/cargopythonpackage.py

```diff
@@ -49,8 +49,7 @@ def extra_options(extra_vars=None):
 
     def __init__(self, *args, **kwargs):
         """Constructor for CargoPythonPackage easyblock."""
-        Cargo.__init__(self, *args, **kwargs)
-        PythonPackage.__init__(self, *args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
This looks very sensible, but I seem to remember there was a reason why we didn't do it like this initially...

@Micket Do you recall why we implemented it like this?